data/huion-h1060p.tablet

```diff
@@ -0,0 +1,24 @@
+# HUION
+# H1060P
```

Comment:
missing the sysinfo link (see e.g. commit 9d61a6c0cf4c0052a53afa288a4bf900e4eb2018) for how to add this, other than that this looks good, thanks!